OtpCore/OtpAuthUri.cs

```diff
@@ -93,6 +98,25 @@ public OtpAuthUri(Uri uri)
                 Parameters[key.ToString().ToLower()] = nameValues[key.ToString()];
             }
 
+            if (Issuer != null && (Issuer.Contains(":") || Account.Contains(":")))
```

Comment:
At this point in time, `Issuer` will never contain a colon, correct? All you have done above is:
```
var split = Label.Split(new[] { ':' }, 2);
Issuer = split[0];
```

So `Issuer` is any text before the first colon. Therefore, it will never contain a colon itself, correct?